website/content/plugins/drivers/podman.mdx

```diff
@@ -191,6 +209,13 @@ The `podman` driver implements the following [capabilities](/nomad/docs/concepts
   }
   ```
 
+- `cpu_hard_limit` (Optional) `true` or `false`. Use hard CPU limiting instead of
+  soft limiting. By default this is `false` which means soft limiting is used and
+  containers are able to burst above their CPU limit when there is idle capacity.
```

Comment:
```suggestion
- `cpu_hard_limit` (Optional) `true` or `false`. Use hard CPU limiting instead
  of soft limiting. By default this is `false`, which means Podman uses soft
  limiting so that containers are able to burst above their CPU limit when there
  is idle capacity.
```

Style: active voice and a bit of clarification